src/app.ts

```diff
@@ -5,35 +5,40 @@ import AppdataSource from "./db/data-source";
 import errorMiddleware from "./middleware/error.middleware";
 import departmentRouter from "./routes/department.routes";
 import cors from "cors";
+import bookDetailRouter from "./routes/bookDetail.routes";
+import booksRouter from "./routes/books.routes";
+import ShelfRouter from "./routes/shelf.routes";
 
 const server = express();
 server.use(loggerMiddleware);
 server.use(express.json());
 server.use(
-	cors(
-	// 	{
-	// 	origin: "http://localhost:5173",
-	// 	credentials: true,
-	// }
-	)
+    cors()
+    // 	{
+    // 	origin: "http://localhost:5173",
+    // 	credentials: true,
+    // }
 );
 server.use("/employee", employeeRouter);
 server.use("/department", departmentRouter);
+server.use("/books", bookDetailRouter);
```

Comment:
better to have the entity name to avoid confusion ! something like /book-details